otx/algorithms/common/adapters/mmcv/models/backbones/torchvision_backbones.py

```diff
@@ -0,0 +1,247 @@
+"""Torchvision Model Backbone Class Generation.
+
+For torchvision backbone support for OTX models,
+this is a code that converts torchvision backbone classes to match
+the mmcv backbone class format and registers them in the mmcv registry.
+This copied the format of "mmdet/models/backbones/imgclsmob.py"
+as it is and made some modifications & code cleaning.
+"""
+# Copyright (C) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import inspect
+import os
+
+from mmcv.cnn import build_activation_layer, build_norm_layer
+from torch import nn
+from torch.nn.modules.batchnorm import _BatchNorm
+from torchvision import models
+from torchvision.models.utils import load_state_dict_from_url
+
+from ..builder import TORCHVISION_BACKBONES
+
+# pylint: disable=protected-access, assignment-from-no-return, no-value-for-parameter, too-many-statements
+
+
+def get_torchvision_models():
+    """Get torchvision backbones of current version."""
+    torchvision_urls = {}
+    torchvision_models = {}
+    for model_key, model_value in models.__dict__.items():
+        if callable(model_value) and model_key[0].islower() and model_key[0] != "_":
+            torchvision_models[model_key] = model_value
+        elif inspect.ismodule(model_value) and hasattr(model_value, "model_urls"):
+            torchvision_urls.update(model_value.model_urls)
+    return torchvision_models, torchvision_urls
+
+
+TORCHVISION_MODELS, TORCHVISION_MODEL_URLS = get_torchvision_models()
+
+
+def replace_activation(model, activation_cfg):
```

Comment:
This can be used on the configuration of the backbone. Depending on the value of `activation_cfg`, it goes through this function.

Reply:
I see the code, but does any of existing configuration files use that functionality? Is it possible to update this config via CLI and will users do that?
This function looks like a workaround to avoid using Hardswish activations, because we used to have problems with exporting swish to onnx in older opsets. Overall, it's better to avoid comparisons like `name == "activ"` even in such hacky wrappers, because they are easy to brake.

Reply:
Thanks for the nice comment! This was a code copied from the method used when registering the pytorchcv backbone in `mmdetection`. I didn't know the details...
Does this mean that this function is now that can be removed? If not sure, I'll edit the comparison as per your comment. :)

Reply:
I tried to change the comparisons as you suggested, but I have no particular idea. Is there any way you can suggest?

Reply:
I guess so, we can safely remove that workaround. It silently replaces swish activations, which basically is not expected by users.

Reply:
Yes i understood!
but, could I create a separate PR to remove and test it?
(There are things that need to be manually tested locally.)

Reply:
I made a jira [ticket](https://jira.devtools.intel.com/browse/CVS-98625) to check this. I'll take the time later to make a related PR and request a review!